.gitlab-ci.yml

```diff
@@ -256,23 +256,18 @@ before_script:
     OPAM_SWITCH: "edge"
     OPAM_VARIANT: "+flambda"
 
-.windows-template:
+.platform-template:
   stage: stage-1
   interruptible: true
+  variables:
+    PLATFORM: "https://github.com/coq/platform/archive/master.zip"
   artifacts:
-    name: "%CI_JOB_NAME%"
+    name: "$CI_JOB_NAME"
```

Comment:
Incorrect change afaict, if you click on download eg at https://gitlab.com/coq/coq/-/jobs/942269583 the filename is `_CI_JOB_NAME.zip`